src/backend/marsha/core/tests/test_models_deletion.py

```diff
@@ -493,10 +492,7 @@ def test_playlist_access_uniqueness(self):
             user=user,
         )
 
-    @mock.patch.object(
-        BaseModel, "_get_conditional_non_deleted_unique_checks", return_value=[]
-    )  # we return nothing on our own method to let django do the default unique validation only
-    def test_django_default_uniqueness(self, mock_checks):
+    def test_django_default_uniqueness(self):
         """Verify that by default Django ignores our unique-together."""
```

Comment:
This test seems useless (and counter demonstrative?) because Django's `UniqueContraint` class does not go through the `validate_unique` method...